### PR TITLE
Participant accordion

### PR DESCRIPTION
**Pull request by jefiedler**
I have created two accordions. One which is used to create or edit the participants. Furthermore I have created an accordion for the display of data.

https://material-ui.com/components/accordion/

### CODE REVIEW

File: client/src/stories/Component/Accordion/ShowParticipant/ShowParticipant.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <Grid container item xs={12} spacing={3}>
+              <TextOutput idName="salutation" title="Anrede" data="Herr" />
+            </Grid>
+            <Grid container item xs={12} spacing={2}>
+              <TextOutput idName="lastName" title="Nachname" data={lastName} />
+              <TextOutput idName="firstName" title="Vorname" data={firstName} />
+              <DateOutput idName="birth" label="Geburtsdatum" data={birth} />
+            </Grid>
+            <Grid container item xs={12} spacing={2}>
+              <TextOutput idName="street" title="Straße" data="" />
```
Comment (lmachens) [inline client/src/stories/Component/Accordion/ShowParticipant/ShowParticipant.js:55]: What about these empty `data`?

**Review by lmachens — APPROVED** [review]
lgtm 🎉

File: client/src/stories/Component/Accordion/EditParticipant/EditParticipantAccordion.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  return (
+    <Grid container spacing={3}>
+      <Accordion
+        expanded={expanded === "panel1"}
+        onChange={handleChange("panel1")}
+      >
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel1bh-content"
+          id="panel1bh-header"
```
[the remark this answers is not in the document]
Comment (jefiedler) [reply client/src/stories/Component/Accordion/EditParticipant/EditParticipantAccordion.js:36]: I think I do not need the IDs. I have read the documentation of the accordion again and did not find the ID explezip. So I think I will take them out.

File: client/src/stories/Component/Accordion/ShowParticipant/ShowParticipant.js (hunk 1 above)
Comment (jefiedler) [reply client/src/stories/Component/Accordion/ShowParticipant/ShowParticipant.js:55]: Yes I need the `data` there. The `data` should be loaded later from the database. The `data=" "` is the placeholder at the moment. I know that this is not the best way.

**Pull request merged**